src/en/articles/font-size-and-colors.md

```diff
@@ -13,20 +14,26 @@ In this article, we will try to answer the following questions concerning the ac
 - <a href="https://www.w3.org/TR/WCAG22/#resize-text"> 1.4.4 Resize text </a>
 - <a href="https://www.w3.org/TR/WCAG22/#contrast-minimum"> 1.4.3 Contrast (Minimum) </a> 
 
 ### Text size enlargement
 
-If the guidelines do not impose a minimum size for characters, [criterion 1.4.4](https://www.w3.org/TR/WCAG22/#resize-text) indicates that the user must be able to increase text size up to 200% without loss of content or functionality. To comply with this criterion, it is essential to test. The procedure to increase text size is available on the following page: [Text size enlargement](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/text-zoom/).
-Sometimes the sizing of certain blocks of text, in particular using sizes in pixels, can lead to loss of information (truncated texts), it is therefore advisable to use relative units (%, em, rem ...). 
+If the guidelines do not impose a minimum size for characters, [criterion 1.4.4](https://www.w3.org/TR/WCAG22/#resize-text) indicates that the user must be able to increase text size up to 200% without loss of content or functionality. To comply with this criterion, it is essential to test. The procedure for increasing text size is available on the following page: [Text Size Enlargement](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/text-zoom/).
+Sometimes, the sizing of certain blocks of text, particularly when using pixel sizes, can lead to the loss of information (truncated or overlapped texts). It is therefore advisable to use relative units (%, em, rem, etc.). 
+
+The method for increasing text size is available here: [Text size enlargement](/en/web/toolbox/methods-and-test-tools/text-zoom/).
+
+**Note:** at Orange, we decided to use 200% text only size enlargement to evaluate this criterion and not by the other two possible methods. We believe that this provides greater comfort for users, even if it requires more work for developers.
+
+
 
 ### Color contrast
 
-The guidelines do not require the use or even prohibit the use of certain colors for texts. However, a light gray text on a white background, for example, could be difficult. It is therefore essential to check that the text color and the background color provide a sufficient level of contrast (see levels below). This can be done easily with help [of tools](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/color-contrast-level/).
+The guidelines do not require the use of, or even prohibit the use of, certain colors for text. However, a light gray text on a white background, for example, could be difficult. It is therefore essential to check that the text color and the background color provide a sufficient level of contrast (see levels below). This can be done easily with help [of tools](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/color-contrast-level/).
```

Comment:
```suggestion
The guidelines do not require or prohibit the use of certain colors for text. However, a light gray text on a white background, for example, could pose difficulties. It is therefore essential to check that the text color and the background color provide a sufficient level of contrast (see levels below). This can be done easily with help [of tools](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/color-contrast-level/).
```